xml/System/Span`1.xml

```diff
@@ -130,11 +153,20 @@
         <Parameter Name="length" Type="System.Int32" />
       </Parameters>
       <Docs>
-        <param name="array">To be added.</param>
-        <param name="start">To be added.</param>
-        <param name="length">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="array">The source array.</param>
+        <param name="start">The index of the first element to include in the new <see cref="T:System.Span~1" />.</param>
```

Comment:
Is using the tilde intentional? `T:System.Span~1` In other places we use `T:System.Span``1`

Reply:
Good catch @ahsonkhan. No, they're not by design.